OpenSim/Sandbox/ExampleHopperDevice/exampleHopperDevice.cpp

```diff
@@ -63,7 +63,7 @@ static const std::string shankAttachment{"/Dennis/?????"}; //fill this in
 //      muscle is active. To do this, we will need to connect the vastus
 //      muscle's "activation" output to the controller's "activation" input.
 // [Step 3, Task B]
-static const std::string vastusActivationOutput{"/Dennis/?????"}; //fill this in
+static const std::string vastus{"/Dennis/?????"}; //fill this in
```

Comment:
The comment suggests that this string should point to the vastus muscle's "activation" output (as it was originally), but the variable name seems to contradict this interpretation.